consul_config.py.ctmpl

```diff
@@ -70,6 +70,11 @@ MBID_MAPPING_DATABASE_URI = "dbname=musicbrainz_json_dump user=musicbrainz host=
 MBID_MAPPING_DATABASE_URI = ""
 {{end}}
 
+# for use in playlists admin view
+SQLALCHEMY_BINDS = {
+   'timescale': SQLALCHEMY_TIMESCALE_URI
```

Comment:
does this mean that the sqlalchemy timescale url is defined here, but the regular db url is defined in `SQLALCHEMY_DATABASE_URI`? Does it make sense to combine them? I guess that means that we'd have to set `__bind_key__` in all other db models as well?

Reply:
https://flask-sqlalchemy.palletsprojects.com/en/2.x/binds/

The `SQLALCHEMY_DATABASE_URI` one is the default to be used if no other bind is specifically defined. The docs also define one default and then extra ones are in `SQLALCHEMY_BINDS`. I think its fine to keep it this way.